API_README.md

```diff
@@ -196,6 +196,25 @@ curl -X POST 'http://localhost:8300/apis/api/patient/1/encounter' -d \
 }'
 ```
 
+
+#### PUT /api/patient/:pid/encounter/:eid
+
+```sh
+curl -X POST 'http://localhost:8300/apis/api/patient/1/encounter/1' -d \
+'{
+    "date":"2019-09-14",
+    "onset_date": "0000-00-00 00:00:00",
```

Comment:
lets make this:
```
"onset_date": "",
```
(zero dates will be illegal in OpenEMR's database sometime in the future)

Reply:
Will fix this